app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -123,8 +123,10 @@ $include-html-global-classes: $include-html-classes;
 // $warning-color: #f08a24;
 // $info-color: #a0d3e8;
 
+$primary-color: #ffa133;
+
 // We use these to control various global styles
-// $body-bg: $white;
+ $body-bg: #008CBA;
```

Comment:
Color `#008CBA` should be written as `#008cba`<br>Line should be indented 0 spaces, but was indented 1 spaces